README.md

```diff
@@ -1 +1,128 @@
+Queries can combine multiple conditions with keyword AND and OR, and use the keyword NOT to negate a conditions.
+
+## Requirements
+
+ - Python >= 3.10
+
+
+## Dependencies and Limitations
+
+1. When adding tags to topics this agent calls the platform.historian's (or a configured historian's) 
+   get_topic_list and hence requires a platform.historian or configured historian to be running, but it doesn't require 
+   the historian to use sqlite3 or any specific database. It requires historian to be running only for using this 
+   api (add_tags) and does not require historian to be running for any other api. 
+2. Resource files that provides the list of valid tags is mandatory and is present under data_model/tags.csv   
+   within base tagging agent
+3. Tagging agent only provides APIs query for topic names based on tags. Once the list of topic names is retrieved, 
+   users should use the historian APIs to get the data corresponding to those topics. 
+4. Current version of tagging agent does not support versioning of tag/values. When tags values set using tagging 
+   agent's APIs update/overwrite any existing tag entries in the database
+5. Since RDMS is not a natural fit for tagname=value kind of data, performance of queries will not be high if you have 
+   several thousands of topics and several hundreds tags for each topic and perform complex queries. For intermediate 
+   level data and query complexity, performance can be improved by increasing the page limit of sqlite.
+
+## Installation
+
+1. Create and activate a virtual environment.
+
+   ```shell
+    python -m venv env
+    source env/bin/activate
+    ```
+
+2. Installing volttron-sqlite-tagging requires a running volttron instance.
+
+    ```shell
+    pip install volttron
+    
+    # Start platform with output going to volttron.log
+    volttron -vv -l volttron.log &
+    ```
+
+3. Create an agent configuration file 
+   SQLite tagging supports three parameters
+    
+    - connection -  This is a mandatory parameter with type indicating the type of database (i.e. sqlite) and params 
+                    containing the path the database file.
+    
+    - table_prefix - Optional parameter to provide custom table names for topics, data, and metadata.
+   
+    - historian_vip_identity - Optional. Specify if you want tagging agent to query the historian with this vip 
+                               identity. defaults to platform.historian. Historian is queried only by the add_topics api.
+                               other apis do not use historian.
+    
+    The configuration can be in a json or yaml formatted file.
+
+    Yaml Format:
+
+    ```yaml
+    connection:
+      # type should be sqlite
+      type: sqlite
+      params:
+        # Relative to the agents data directory
+        database: "data/tagging.sqlite"
+    
+    # prefix for tagging data tables
+    table_prefix: ""
+
+    historian_vip_identity: my.test.historian
```

Comment:
perhaps add a comment about the historian_vip_identity

Reply:
its right after the documentation for it at line 72

Reply:
my point is we have # lines for every other param, but not this one.  This perhaps should be commented out since its optional?

Reply:
Fixed